DetectorDescription/RecoGeometry/interface/DTNumberingScheme.h

```diff
@@ -0,0 +1,32 @@
+#ifndef GEOMETRY_RECO_GEOMETRY_DT_NUMBERING_SCHEME_H
+#define GEOMETRY_RECO_GEOMETRY_DT_NUMBERING_SCHEME_H
+
+#include "DetectorDescription/DDCMS/interface/MuonNumbering.h"
+
+namespace cms {
+  struct DTNumberingScheme {
```

Comment:
`class` please

Reply:
`DTNumberingScheme` seems complex enough to qualify as a class.